src/modules/profile/components/ImagePicker/ImagePicker.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+import {TouchableOpacity} from 'react-native';
+import * as ExpoImagePicker from 'expo-image-picker';
+import {StyledImagePickerWrapper, Avatar} from "./ImagePicker.styled";
+import defaultPicture from '@assets/profile-default.png';
+
+interface ImagePickerProps {
+    imageHandler: (image: string) => void,
+    value: string
+}
+
+const ImagePicker = (props: ImagePickerProps) => {
+
+    const {
+        imageHandler = () => null,
+        value: defaultValue = defaultPicture
+    } = props;
+
+    const [image, setImage] = useState<string>('');
+    useEffect(() => {
+        (async () => {
+            const { status } = await ExpoImagePicker.requestMediaLibraryPermissionsAsync();
+            if (status !== 'granted') {
+                alert('Sorry, we need camera roll permissions to make this work!');
```

Comment:
It shouldn't be shown from the start, only when user pressed on icon. Also, what if user hasn't granted the permission? You will just say `we need it`? You can find better way in my lection.